lib/optionsSchema.json

```diff
@@ -145,6 +145,10 @@
       "description": "The passphrase to a (SSL) PFX file.",
       "type": "string"
     },
+    "requestCert": {
+      "description": "Enables request for client certificate.",
```

Comment:
Please add "This is passed directly to the https server" to the description so folks know what the purpose is, since there are some mixed-use child properties in the `https` property.